plugins/web/opentelemetry-instrumentation-user-interaction/src/instrumentation.ts

```diff
@@ -320,7 +320,7 @@ export class UserInteractionInstrumentation extends InstrumentationBase<unknown>
             return plugin._invokeListener(listener, this, args);
           }
         };
-        if (plugin.addPatchedListener(this, type, listener, patchedListener)) {
+        if (listener && plugin.addPatchedListener(this, type, listener, patchedListener)) {
```

Comment:
Wouldn't this be taken care of [few lines above](https://github.com/open-telemetry/opentelemetry-js-contrib/blob/f8cabf306faa3e3eb4b9ce38ccdde842abdb2b82/plugins/web/opentelemetry-instrumentation-user-interaction/src/instrumentation.ts#L288)? seems the functions starts with 

```js
        // Forward calls with listener = null
        if (!listener) {
          return original.call(this, type, listener, useCapture);
        }
``` 

So I would expect that when hitting the current line `listener` is tested to be truthy